### ISSUE TITLE
Mqtt Sink - dynamic topic - multiple variables

### ISSUE THREAD

**Issue by CrystalMethod** [lensesio/stream-reactor#589]
Would it be considerable to extend the mechanism with the dynamic topics (#491) to multiple variables? Would  like to write something as follows:

```
"INSERT INTO `site/$site/line/$line/robot/$robot` SELECT * FROM ... WITHTARGET = site, line, robot"
```
Does anyone have a better approach?

**Comment by andrewstevenson** [comment]
@CrystalMethod I think this just builds on #491 so LGTM, are you willing to submit a PR?

**Comment by CrystalMethod** [comment]
Let me see how far I can get with that.

**Comment by andrewstevenson** [comment]
Fixed in #650
